packages/web-components/fast-foundation/src/avatar/stories/avatar.register.ts

```diff
@@ -0,0 +1,107 @@
+import { attr, css, html, when } from "@microsoft/fast-element";
+import { DesignSystem } from "../../design-system/design-system.js";
+import { Avatar as FoundationAvatar } from "../avatar.js";
```

Comment:
There are two cases where having the class called something else would be useful. 

Case 1: to disambiguate between the extended class and its base class. Only a few components still do this (`avatar`, `horizontalscroll`, `listbox`, and `slider`), and it's because their stories would break without some extra assistance. Here, the avatar element wouldn't be able to support images without a class extension since the only thing declaring the `media` option is the `AvatarOptions` type - there's no additional logic present in the template for this feature.

Case 2: to disambiguate between the component class and its default storybook story export. Storybook likes consistency, so when the default export's `title` matches the variable name of one of the named exports, it connects the two and sets it up to match that name. All other exports present in the stories file are translated from PascalCase to Space Case unless its `storyName` property is specified. Since we want the story to be called Avatar, we have to alias the `Avatar` class type to something different.

So to answer your question, calling everything `Foundation<name>` would certainly help with both of these cases.

Reply:
Cool. Assuming @chrisdholt is ok with that, we can make those renames as part of the next set of changes.

Reply:
That change would also be helpful in the CLI.